### DISCUSSION TITLE
Slack invite not working

### DISCUSSION THREAD

**Discussion by aalsaleh** [Kong/kong#11681] [Help]
Hi

The Slack Invite Request Link (https://join.slack.com/t/kongcommunity/shared_invite/zt-1s4nb74vo-jLdMEk8MoTm~uMWYMMLPWg) that is on https://konghq.com/community appears to be broken.

Is there a new link or another way to access the Kong Community Slack Channel?

Thanks

**Comment by StarlightIbuki** [comment]
I've created a link: https://join.slack.com/t/kongcommunity/shared_invite/zt-26b3klqqo-sTqLGPgZM~JwSgzsUkN3nA
And I'm looking for the right place to request for change to that page.

**Comment by lindeskar** [comment]
This is still true!
